_pytest/config.py

```diff
@@ -977,6 +977,9 @@ def _initini(self, args):
         self.invocation_dir = py.path.local()
         self._parser.addini('addopts', 'extra command line options', 'args')
         self._parser.addini('minversion', 'minimally required pytest version')
+        self._parser.addini('make_parameterize_legacy',
```

Comment:
i fear we will have to do some very pedantic nitpicking over the option name,

after reading trough it a few time i realize its painstakingly hard to find an name fitting the context and not about 4 lines long

my entirely personal opinion is to still lengthen it to something like `make_parameterize_id_legacy_mode`
but i'd like to hear what others think as well